play-media-native/hooks/useFacets/useFacets.ts

```diff
@@ -32,3 +33,27 @@ export const useFacets = ({
 
   return filteredItems;
 };
+
+export const useSearchFacets = ({
+  initialItems,
+  facets,
+  query,
+}: {
+  initialItems: Record<string, any>[];
+  facets: IIndexable;
+  query: string;
+}) => {
+  const finalItems = useMemo(() => {
```

Comment:
I'm getting a bit of a strange behavior when filtering using the search input. Even though the items at the top seem correct, there are also some items in the list that do not seem relevant at first sight. Maybe some refinement needed here?